common/test/java/com/couchbase/lite/DocumentTest.java

```diff
@@ -2712,6 +2712,16 @@ public void testMutableFromArray() {
             () -> new MutableDocument("fromJSON", BaseDbTestKt.readJSONResource("array.json")));
     }
 
+    // Unsupported revision history
+    @Test
+    public void testGetRevisionHistory() throws CouchbaseLiteException {
+        MutableDocument mdoc = new MutableDocument("doc1");
+        Collection coll = getTestCollection();
+        coll.save(mdoc);
+        Document doc = coll.getDocument(mdoc.getId());
+        assertTrue(doc.getRevisionHistory().startsWith("1-"));
```

Comment:
I don't think this is correct for version vector. Probably just testing that it's not null and length > 0?